spring-boot-samples/spring-boot-sample-cache/src/main/resources/infinispan.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <infinispan xmlns="urn:infinispan:config:7.2">
 
-	<cache-container default-cache="default">
+	<cache-container default-cache="countries">
```

Comment:
I don't think having `countries` as the default cache is a good idea for the sample. None of the other samples do that. I've already polished that part in https://github.com/spring-projects/spring-boot/commit/fc38c1ba4d242ee1bfdb44b85c84e53276632d3d based on your feedback so we should be good already.